resources/Account.js

```diff
@@ -54,13 +54,14 @@ class Account extends EventEmitter {
             .then((siteUserData) => {
                if (siteUserData) {
                   try {
-                     this.username = siteUserData.user.username;
+                     this.username = siteUserData?.user?.username;
                      if (this.username != "??") {
                         analytics.setUserName(this.username);
                      }
-                  }
-                  catch (err) {
-                     console.error("Couldn't read username from stored data");
+                  } catch (err) {
+                     console.error({
+                        "Couldn't read username from stored data": data
+                     });
                      console.error("Do we need it?");
                      console.error(err);
                   }
```

Comment:
Just one minor point: The try/catch was only there in case `siteUserData.user.username` triggered an error due to user being undefined. Using `siteUserData?.user?.username` prevents that error altogether so there's now nothing to catch.